extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/AuthRuntimeConfig.java

```diff
@@ -25,6 +27,15 @@ public class AuthRuntimeConfig {
     @ConfigItem(name = "policy")
     public Map<String, PolicyConfig> rolePolicy;
 
+    /**
+     * Add roles granted to the `SecurityIdentity` based on the roles that the `SecurityIdentity` already have.
```

Comment:
I propose to simplify (or reword) it to something like this: 

```
Map the `SecurityIdentity` roles to deployment specific roles and add the matching roles to  `SecurityIdentity`.
<p/>
For example, if `SecurityIdentity` has a `user` role and the endpoint is secured with a 'UserRole' role, use this property to map the `user` role to the `UserRole` role, and have `SecurityIdentity` to have both `user` and `UserRole` roles.
```

I'm not sure we should be referring to OIDC from this property's java docs